docs/sdks/js-ts/js-ts-apis.mdx

```diff
@@ -70,10 +70,12 @@ type Entries = { [key: string]: string | number | null };
 
 Calls a `view` action on the network. If authentication for the view action is enforced by a Kwil Gateway, a `kwilSigner` must be provided. The call method will prompt the user to authenticate if they have not already done so, and will retain a cookie to authenticate with the Kwil Gateway for all subsequent calls.
 
+A `kwilSigner` is also required if the Kwil network is running in Private mode.
+
 #### Parameters
 
 -   `actionBody`: The message to call. This object should match the `ActionBody` interface and contain the action name and database ID. The `inputs` array must only have one entry.
--   `kwilSigner` (optional): The kwilSigner for the call request. This is only required if the action uses a `@caller` contextual variable. The SDK will use the `identifier` in the `kwilSigner` to fill in the `@caller` contextual variable.
+-   `kwilSigner` (optional): The kwilSigner for the call request. This is only required if the action uses a `@caller` contextual variable or the Kwil network is running in Private mode. The SDK will use the `identifier` in the `kwilSigner` to fill in the `@caller` contextual variable.
```

Comment:
While it is certainly the intent that the entire network (all nodes in the network) run with private RPC mode enabled, this is more accurately described as the Kwil node running in private mode.

Reply:
This comment is still not resolved.

Reply:
![Screenshot 2024-10-11 at 3 03 24 PM](https://github.com/user-attachments/assets/88fa2195-65fd-4c54-a532-73990612cb83)

@KwilLuke